drivers/flash/spi_nor.c

```diff
@@ -1159,6 +1289,23 @@ static int spi_nor_process_bfp(const struct device *dev,
 
 	LOG_DBG("Page size %u bytes", data->page_size);
 
+#ifdef CONFIG_SPI_NOR_SUSPEND_RESUME
+	if (php->len_dw >= 13) {
+		uint32_t dw12 = sys_le32_to_cpu(bfp->dw10[2]);
```

Comment:
there can be devices, that don't support reading in the suspended area of program or erase. You might want to check that from the bfp.